app/resources/base.html.template

```diff
@@ -75,92 +75,85 @@
           {% block logo %}{% endblock %}
         {% endif %}
       {% endblock header %}
-
-      {% if env.repo %}
-        <a class="repo" href="{{env.repo_title_url}}" {{env.target_attr|safe}}>
-          {{env.repo_title}}
-        </a>
-      {% endif %}      
-
     </div>
 
+    {% if env.repo %}
+      <div class="subtitle-bar">
+        {{env.repo_title}}
+      </div>
+    {% endif %}
 
     {% if env.repo_test_mode %}
       <div class="header-notification">
         {% trans "This repository is currently in test mode. While test mode is in effect, records that are over 24 hours old are deleted." %}
       </div>
     {% endif %}
 
-    {% block container %}
-      <div class="container">
-        <div class="content" role="main">
-          {% block content %}
-          {% endblock content %}
+    <div class=window>
```

Comment:
Enclose the class name with double-quotes.

Also, "window" may be confusing because usually the whole page is called "window" in HTML. I don't have a good naming idea, but maybe something like "outer-container", "content-pane". (We actually want to say "everything except for the header and the side bar", but that would be too long.)